src/Plugin/Purl/Method/PathPrefixMethod.php

```diff
@@ -24,11 +24,23 @@ private function pathContains($modifier, $path)
         return strpos($path, '/' . $modifier) === 0;
     }
 
+    /**
+     * Allow for altering the request when the RequestSubscriber event fires.
+     *
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param $identifier
+     *
+     * @return \Symfony\Component\HttpFoundation\Request
```

Comment:
Looks like the doc standards suggest we should have |false at the end of this, to indicate the other type of return value...  https://www.drupal.org/node/1354#types and https://www.drupal.org/project/drupal/issues/711918